frontend/src/main/pages/HomePage.js

```diff
@@ -4,14 +4,14 @@ import BasicCourseSearchForm from "main/components/BasicCourseSearch/BasicCourse
 import { useBackendMutation } from "main/utils/useBackend";
 import { queryAllByTestId } from "@testing-library/react";
 import { toast } from "react-toastify";
+import BasicCourseTable from "main/components/Courses/BasicCourseTable";
 
 export default function HomePage() {
   // Stryker disable next-line all : Can't test state because hook is internal
-  const [courseJSON, setCourse] = useState([]);
+  var [courseJSON, setCourseJSON] = useState([]);
```

Comment:
Aren't you supposed to use `const` with `useState` (instead of `var`)?

Reply:
usually yes, but I don't think it affects much in our case

Reply:
I changed it because using `var` goes against the convention and allows us to modify `courseJSON` directly without using `setCourseJSON` which could mess up the rendering if I understand React correctly